esrally/metrics.py

```diff
@@ -1821,6 +1830,7 @@ def add_op_metrics(self, task, operation, throughput, latency, service_time, pro
             "service_time": service_time,
             "processing_time": processing_time,
             "error_rate": error_rate,
+            "duration_time": duration
```

Comment:
I'd argue that `duration` by itself conveys its meaning already and the `_time` suffix is redundant? (see also https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/time/Duration.html)